projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_calendar.scss

```diff
@@ -198,9 +183,9 @@ $_light-calendar: extend(
 
 /// Generates a fluent calendar schema.
 /// @type {Map}
-/// @prop {Color} date-current-text-color [#fff] - The text color for the current date.
+/// @prop {Color} date-current-text-color [igx-color: 'surface'] - The text color for the current date.
 /// @prop {Map} date-current-bg-color [igx-color: ('primary', 500)] - The background color for the current date.
-/// @prop {Map} date-selected-background [igx-color: ('primary', 100), rgba: .5, to-opaque: #fff] - The background color for the selected date.
+/// @propsivanov-paletteColorThemes {Map} date-selected-background [igx-color: ('primary', 50)] - The background color for the selected date.
```

Comment:
what is it with this one @simeonoff my branch name appears appended to the end of the prop